tests/e2e/utils/mock-requests.js

```diff
@@ -11,31 +11,56 @@ export default class MockRequests {
 		this.page = page;
 	}
 
+	/**
+	 * Fulfill a request multiple times.
+	 *
+	 * @param {number} times The number of times to fulfill the request.
+	 * @return {Proxy} A proxy object to modify the behavior of the `fulfillRequest` method.
```

Comment:
Suggested to change to a more precise type and description:

```suggestion
	 * @return {this} A proxied instance intercepts the subsequent fulfillRequest calls to attach the `times` option.
```